Not able to install new packages easily

After the recent merge of `pnpm-workspace` everytime i want to install a new package it is followed by a warning and i need to use -w flag everytime

```
sjain@Sajals-MacBook-Pro FlowTest % pnpm install brace       
 ERR_PNPM_ADDING_TO_ROOT  Running this command will add the dependency to the workspace root, which might not be what you want - if you really meant it, make it explicit by running this command again with the -w flag (or --workspace-root). If you don't want to see this warning anymore, you may set the ignore-workspace-root-check setting to true.
```

cc: @xelacast

--- a/.npmrc
+++ b/.npmrc
@@ -1,1 +1,2 @@
 node-linker=hoisted #pnpm requirement when working with electron
+ignore-workspace-root-check=true

